modules/wpcom-block-editor/functions.classic-editor.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * This file contains some 'remember' functions taken from the core Classic Editor Plugin
+ * Used to align the 'last editor' metadata so that it is set on all Jetpack and WPCOM sites
+ *
+ * @package Jetpack
+ */
+
+namespace Jetpack\ClassicEditor;
+
+/**
+ * Remember when the classic editor was used to edit a post.
+ *
+ * @param object $post The post being editted.
+ */
+function remember_classic_editor( $post ) {
+	$post_type = get_post_type( $post );
+
+	if ( $post_type && post_type_supports( $post_type, 'editor' ) ) {
+		remember_editor( $post->ID, 'classic-editor' );
+	}
+}
+
+/**
+ * Remember when the block editor was used to edit a post.
+ *
+ * @param  object $editor_settings This is hooked into a filter and this is the settings that are passed in.
+ * @param  object $post            The post being editted.
+ * @return object                  The unmodified $editor_settings parameter.
+ */
+function remember_block_editor( $editor_settings, $post ) {
+	$post_type = get_post_type( $post );
+
+	if ( $post_type && classic_editor_can_edit_post_type( $post_type ) ) {
+		remember_editor( $post->ID, 'block-editor' );
+	}
+
+	return $editor_settings;
+}
+
+/**
+ * Sets the metadata for the specified post and editor.
+ *
+ * @param int    $post_id The ID of the post to set the metadata for.
+ * @param string $editor  String name of the editor, 'classic' or 'gutenberg'.
```

Comment:
Should we update this to use the correct strings, `classic-editor` and `block-editor`?

```suggestion
 * @param string $editor  String name of the editor, 'classic-editor' or 'block-editor'.
```

Reply:
Ah, nice spot, thanks!